packages/babel-plugin-transform-runtime/test/fixtures/regression/with-env-useBuiltIns/output.js

```diff
@@ -0,0 +1,7 @@
+var _createClass = require("@babel/runtime/helpers/createClass");
```

Comment:
This test is to make sure that when using `preset-env` to load polyfills, and when using `transform-runtime` just for helpers, the helpers are loaded from the package without polyfills (for backwards compatibility).